README.md

```diff
@@ -69,6 +70,33 @@ jobs:
     - name: Cache workaround # https://github.com/actions/cache/issues/133
       run: chmod -R a+rwx .
 ```
+You can also use the provided Github Action:
+```
+name: GitHub Action sample
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build_test_and_deploy:
+    runs-on: ubuntu-18.04 # Works also with self hosted runner supporting docker
+    container:
+        image: docker://docker:stable-git
+
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v2.1.0
+
+    - name: Test action
+      uses: fabernovel/docker-android
+      id: docker-android-action
+      with:
```

Comment:
We should add the tag parameter in example as it might be important to define which docker image to use here.